src/populators/users.populator.js

```diff
@@ -0,0 +1,11 @@
+module.exports = (() => {
+  function populate(users) {
+    if (!users) {
+      return [];
+    }
+    const UserPopulator = require('./user.populator.js');
```

Comment:
should we move require outside of function?

Reply:
They can be in the function.